app/js/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default class Button extends React.PureComponent {
```

Comment:
I'll keep the custom component and make the adjustments you recommended in the PR.